doc/cephadm/monitoring.rst

```diff
@@ -93,6 +93,39 @@ completed, you should see something like this from ``ceph orch ls``
   node-exporter      2/2  6s ago     docker.io/prom/node-exporter:latest             e5a616e4b9cf  present
   prometheus         1/1  6s ago     docker.io/prom/prometheus:latest                e935122ab143  present
 
+Configuring SSL/TLS for Grafana
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+By default, cephadm deploys Grafana with a self-signed certificate which is
+generated just when the service is deployed and no custom certificate has been
+specified. This, of course, may not be optimal. The users would need to accept
+an invalid certificate. You may also have a custom certificate that is accepted
+internally which you want to deploy. In either case, you might want to set a
+custom certificate for the deployment of Grafana. To deploy Grafana with a
+custom certificate, issue the following commands.
+
+.. code-block:: bash
+
+  ceph config-key set mgr/cephadm/grafana_key -i $PWD/key.pem
+  ceph config-key set mgr/cephadm/grafana_crt -i $PWD/certificate.pem
+
+The ``cephadm`` manager module needs to be restarted to be able to read updates
+to these keys.
+
+.. code-block:: bash
+
+  ceph orch restart mgr
```

Comment:
```suggestion
    ceph orch restart mgr

Reply:
World's smallest nit - aligning  the "c" from ceph, with the "c" from code. Ignore or otherwise. Just a preference.